src/main/java/impl/chatbot/Danny.java

```diff
@@ -0,0 +1,41 @@
+package impl.chatbot;
+
+import impl.storage.TaskList;
+import impl.ui.Parser;
+import impl.ui.Ui;
+
+import java.util.Scanner;
```

Comment:
I think you need to put all the import java.* statement above import other.*
![Screenshot 2024-09-05 165433](https://github.com/user-attachments/assets/1c4e7662-8d6e-431b-8694-40367b0bf072)